src/components/ButtonTabs/ButtonTabs.tsx

```diff
@@ -19,10 +22,20 @@ export interface ButtonTabsProps {
     items: ButtonTabsItemProps[];
     activeTab?: string;
     onSelectTab?: (tabId: string) => void;
+    blockName?: string;
 }
 
 const ButtonTabs = (props: ButtonTabsProps) => {
-    const {className, items, activeTab, onSelectTab} = props;
+    const {className, items, activeTab, onSelectTab, blockName = COMPONENT_NAME} = props;
+    const defaultEvent = useMemo(
+        () => ({
+            name: 'button-tabs-click',
```

Comment:
i think we need a type, describing all of these events

Reply:
Actually, `type` is coming to the PR in order to organize analytics handler.
There are a few types of events which should be treated slightly different than usual button click (in code of the `page-constructor`) (player click for example).
On the other hand, we want to provide `page-constructor` users with instrument to separate events. They users can use `type` to manage different events in a different way